pkg/local_object_storage/metabase/control.go

```diff
@@ -138,21 +138,31 @@ func (db *DB) init(reset bool) error {
 			if err != nil {
 				return fmt.Errorf("could not sync object counter: %w", err)
 			}
+		} else {
```

Comment:
just a little suggestion: if now we have both branches (if-else) i would prefer not to use negation (`!reset`), i think it would be read simpler